### ISSUE TITLE
Restore this repo

### ISSUE THREAD

**Issue by cburbridge** [strands-project/scitos_robot#68]
I think we should re-instate this repository. The `scitos_pc_monitor` package belongs here as does a basic generic `scitos_bringup`. As the strands robots are not just scitos robots, we should then have another repo `strands_robot` that contains the `strands_bringup` package. I don't think `scitos_bringup` belongs in `scitos_drivers`. 

`scitos_drivers` now no longer has a laser driver, and probably will eventually not have a ptu driver either since it is not a scitos specific device. So maybe we could move `scitos_mira` into this repo and remove `scitos_drivers`.

**Comment by marc-hanheide** [comment]
so, why then not moving scitos_pc_monitor into scitos_drivers and then probably rename it?

**Comment by cburbridge** [comment]
Yes that would be the same, is it ok with everyone (@hawesie) if I do this then?

**Comment by hawesie** [comment]
:+1:
